woocommerce/woocommerce-gutenberg-products-block
# Product Collection: Add Taxonomy filters to sidebar settings

> ⚠️ This PR will be merged after PR #9600, as this branch is checked out from the branch of PR #9600.

This PR adds `Taxonomy` filters to sidebar settings.

<img width="283" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/16707866/2e3afa24-f1f6-4947-b3ca-04d214a0b448">
<img width="283" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/16707866/5233c28f-6098-42be-a927-6003c3d050c4">




Fixes #9364 

### Testing

1. Create a new post
2. Add Product Collection block
3. With `Product Collection` block selected, check the sidebar for the Taxonomy filter i.e., product categories, tags etc. These will be displayed as shown in images below:
<img width="283" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/16707866/2e3afa24-f1f6-4947-b3ca-04d214a0b448">
<img width="283" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/16707866/5233c28f-6098-42be-a927-6003c3d050c4">

4. Make sure that you have products with categories and tags. If not, add categories or tags to some products.
5. Now select a product category or tags. After the selection, verify that changes reflect immediately in the Editor view.
6. Save and publish the post. After publishing, you can just navigate to the Frontend and verify that the changes you made in the Editor are accurately reflected.
7. Return to the Editor and validate the `Reset all` functionality of filters:
   - Clicking the `Reset all` button resets your selected filters.
   - The `Reset all` button can be found as shown in image below:
<img width="279" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/16707866/fe6ad8f9-e9f5-4507-822c-8d42a0496ed0">



### WooCommerce Visibility




### Changelog

> Product Collection: Add Taxonomy filters to sidebar settings
